docs/installation/replay.md

```diff
@@ -1,4 +1,4 @@
-# Assemblyline Replay
+# Multiple instances syncing (Replay)
```

Comment:
"Syncing [Assemblyline] Instances (Replay)" a better title?

"Assemblyline" could be dropped since it's within context of our documentation.